subgraph/schema.graphql

```diff
@@ -0,0 +1,224 @@
+type Account @entity {
+  id: ID!
+  asScanner: Scanner
+  bots: [Bot!]! @derivedFrom(field: "owner")
+  scanners: [Scanner!]! @derivedFrom(field: "owner")
+  managerOf: [ScannerManager!]! @derivedFrom(field: "account")
+  botTransferFromEvents: [BotTransfer!]! @derivedFrom(field: "from")
+  botTransferToEvents: [BotTransfer!]! @derivedFrom(field: "to")
+  botUpdatedEvents: [BotUpdated!]! @derivedFrom(field: "by")
+  scannerTransferFromEvents: [ScannerTransfer!]! @derivedFrom(field: "from")
+  scannerTransferToEvents: [ScannerTransfer!]! @derivedFrom(field: "to")
+  scannerManagerEnabledEvents: [ScannerManagerEnabled!]!
+    @derivedFrom(field: "manager")
+}
+
+type Bot @entity {
+  id: ID!
+  owner: Account!
+  enabled: Boolean!
+  disableFlags: Int!
+  metadata: String!
+  chainIds: [BigInt!]!
+  scanners: [Scanner!]
+
+  botTransferEvents: [BotTransfer!]! @derivedFrom(field: "bot")
+  botUpdatedEvents: [BotUpdated!]! @derivedFrom(field: "bot")
+  botEnabledEvents: [BotEnabled!]! @derivedFrom(field: "bot")
+  linkEnabledEvents: [LinkEnabled!]! @derivedFrom(field: "bot")
+}
+
+type Scanner @entity {
+  id: ID!
+  asAccount: Account!
+  owner: Account!
+  enabled: Boolean!
+  disableFlags: Int!
+  managers: [ScannerManager!]! @derivedFrom(field: "scanner")
+  links: [Link!]! @derivedFrom(field: "scanner")
+  stakeThreshold: StakeThreshold! @derivedFrom(field: "scanner")
+  subject: Subject
+  metadata: String!
+  chainId: BigInt!
+  bots: [Bot!]
+
+  scannerTransferEvents: [ScannerTransfer!]! @derivedFrom(field: "scanner")
+  scannerUpdatedEvents: [ScannerUpdated!]! @derivedFrom(field: "scanner")
+  scannerEnabledEvents: [ScannerEnabled!]! @derivedFrom(field: "scanner")
+  scannerManagerEnabledEvents: [ScannerManagerEnabled!]!
+    @derivedFrom(field: "scanner")
+  linkEnabledEvents: [LinkEnabled!]! @derivedFrom(field: "scanner")
+}
+
+type ScannerManager @entity {
```

Comment:
this new object is not cross referenced in scanner.

Question: can there be more than one manager per scanner ? If yes, you need to put a "managers" fields in scanner that cross reference to this. If no, then you need to remove this object and set the manager directly in the Scanner object

Reply:
There is a field managers in scanner that derives from the scanner manager entity

Reply:
But have addressed the question towards Raul to figure if there can be more than 1 manager per scanner